CONTRIBUTING.md

```diff
@@ -51,21 +52,15 @@ submit a new issue or browse through previous issues.
 ## Issues and Labels
 
 The Charmed HPC documentation issue tracker uses a variety of labels to help organize
-and identify issues. Here is a list of some of these labels, and how the maintainers of
+and identify issues. Here are a few of these labels, and how the maintainers of
 Charmed HPC's documentation use them:
 
-* `Type: Bug` - Issues within the Charmed HPC documentation that either produce errors, unexpected behavior, or cause users to encounter unintended errors.
-
-* `Status: Confirmed` - Issues marked `Type: Bug` that have been confirmed to be reproducible on a separate system.
-
-* `Type: Documentation` - Issues for revising or enhancing Charmed HPC's documentation.
+* `docs` - Issues for general revision and enhancement of Charmed HPC's documentation.
 Can also be used for pull requests.
 
-* `Type: Refactor` - Issues that pertain to improving Charmed HPC's documentation.
-
-* `Type: Enhancement` - Issues marked as an agreed upon enhancement to Charmed HPC's documentation. Can also be used for pull requests.
+* `help wanted` - Issues where we need help from the HPC community to solve.
 
-* `Status: Help Wanted` - Issues where we need help from the Ubuntu HPC community to solve.
+* `good first issue` - Issues that the Charmed HPC documentation maintainers have determine are a suitable level for first time documentation contributors. 
```

Comment:
I think we want `determined` here:

```suggestion
* `good first issue` - Issues that the Charmed HPC documentation maintainers have determined to be suitable for first time contributors. 
```

I removed the "documentation" qualifier for contributors because we want everyone to contribute to documentation, not just the "technical authorship" inclined!